stylelint.config.js

```diff
@@ -21,36 +21,24 @@ module.exports = {
   rules: {
     "at-rule-no-unknown": true,
     "at-rule-no-vendor-prefix": true,
-    "block-opening-brace-space-before": "always-multi-line",
     "color-hex-length": "long",
     "color-named": "never",
     "declaration-block-no-duplicate-properties": true,
-    "declaration-block-semicolon-newline-after": "always",
-    "declaration-block-semicolon-newline-before": "never-multi-line",
     "declaration-colon-newline-after": null,
```

Comment:
I think this line can be removed now: https://github.com/stylelint/stylelint/pull/6504/files#diff-44974d399c711111117bc1e8b77a185f69e88f3d68803628ae0e230769cb27c4